README.md

```
@@ -77,14 +78,38 @@ $ npm run start:dev
 $ npm run start:prod
 ```
 
-## API Endpoints
+## API Documentation
 
-### Authentication
+The API is documented using Swagger/OpenAPI. When the application is running, you can access the interactive API documentation at:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)